dbOperations.php

```diff
@@ -63,6 +88,169 @@ public function delete($employeeId) {
 
 	}
 
+	/**
+     *Selects all the details of all the employees
+     *
+     * @access public
+     * @param void 
+     * @return object/boolean
+     */
+	public function selectAllEmployees() {
+		
+        //Get the records of all the employees
+        $selectEmpDetails = "SELECT employee.eid, employee.firstName, employee.middleName,
+            employee.lastName, employee.gender, employee.dob, employee.mobile, employee.landline,
+            employee.email, employee.maritalStatus, employee.employment, employee.employer,
+            employee.photo, commMedium.empId, commMedium.msg, commMedium.email AS comm_email, 
+            commMedium.call, commMedium.any, address.eid, address.type, address.street, address.city ,
+            address.state, address.zip, address.fax FROM employee JOIN commMedium ON 
+            employee.eid = commMedium.empId JOIN address ON  employee.eid = address.eid";
+        
+        $employeeDetails = $this->executeSql($selectEmpDetails);
+        return $employeeDetails;
+	}
+
+    /**
+     *Selects employee details a particular employee using employee id.(and address type when its provided)
+     *
+     * @access public
+     * @param String
+     * @param Integer 
+     * @return object/boolean
+     */
+    public function selectEmployee($employeeId, $addressType = 0) {
+
+        if ( $addressType === 0 ) {
+
+            $selectEmpDetails = "SELECT employee.eid, employee.prefix, employee.firstName, employee.middleName, employee.lastName, employee.gender, employee.dob, employee.mobile, employee.landline, employee.email, employee.password, employee.maritalStatus, employee.employment, employee.employer, employee.note,employee.photo,commMedium.empId, commMedium.msg, commMedium.email AS comm_email, commMedium.call , commMedium.any FROM employee JOIN commMedium ON employee.eid = commMedium.empId WHERE eid =" . 
```

Comment:
Instead of single line this query can be broken into multiple lines for more readability.